### DISCUSSION TITLE
Error creating bodhi linux client image

### DISCUSSION THREAD

**Discussion by decenttp** [ltsp/ltsp#818] [General] [answered]
Hi,

I wanted to create a new client image using Bodhi Linux but while trying to create the image using virtualbox vmdk i am getting wrong fs type, bad option, bad superblock error. See below: 

root@ltsp-server:/srv/ltsp# ltsp image Bodhi
Running: losetup -rP /dev/loop2 /srv/ltsp/Bodhi.img
Running: mount -t tmpfs -o mode=0755 tmpfs /tmp/tmp.iwwC8Fl9CI/tmpfs
Running: mount -t vfat -o ro /dev/loop2p1 /tmp/tmp.iwwC8Fl9CI/tmpfs/0/looproot
Running: mount -t overlay -o upperdir=/tmp/tmp.iwwC8Fl9CI/tmpfs/0/up,lowerdir=/tmp/tmp.iwwC8Fl9CI/tmpfs/0/looproot,workdir=/tmp/tmp.iwwC8Fl9CI/tmpfs/0/work /tmp/tmp.iwwC8Fl9CI/tmpfs /tmp/tmp.iwwC8Fl9CI/root/
mount: /tmp/tmp.iwwC8Fl9CI/root: wrong fs type, bad option, bad superblock on /tmp/tmp.iwwC8Fl9CI/tmpfs, missing codepage or helper program, or other error.
LTSP command failed: mount -t overlay -o upperdir=/tmp/tmp.iwwC8Fl9CI/tmpfs/0/up,lowerdir=/tmp/tmp.iwwC8Fl9CI/tmpfs/0/looproot,workdir=/tmp/tmp.iwwC8Fl9CI/tmpfs/0/work /tmp/tmp.iwwC8Fl9CI/tmpfs /tmp/tmp.iwwC8Fl9CI/root/
Aborting ltsp

Any pointers as to what might be going wrong. 

Thanks

**Comment by alkisg** [comment, marked as answer]
Hi, use a single ext4 partition, and a flat vmdk disk.

Do not use lvm. Do not use encryption. Do not use a .vdi disk.

**Reply by decenttp** [reply]
Thanks that was indeed the issue. Having a single ext4 partition resolved it. 

I however am facing a strange issue after creating the image on client terminals. Clients load fine but after the user logs in it shows a blank desktop with moksha desktop default background which flickers every couple of seconds or so.  I have tried reducing the number of virtual desktops to 1 only and other changes but nothing seems to make any difference.

**Comment by alkisg** [comment]
I did the following quick test:

    wget https://deac-fra.dl.sourceforge.net/project/bodhilinux/6.0.0/bodhi-6.0.0-64-hwe.iso -O /srv/ltsp/images/x86_64.img
    ltsp kernel

Then I booted a client and I was able to login to the Bodhi session without any issues.
Do that ^ test. If it works for you, you did something wrong in your previous try.
If it doesn't work for you, then try on another client, it might be related to hardware/graphics driver issues.
Or create a USB stick with Bodhi live and boot that client with it; if it doesn't work, it's a Bodhi/Linux issue, not an LTSP issue.
